gaphor/diagram/general/metadata.py

```diff
@@ -0,0 +1,206 @@
+from gaphor.core.modeling.properties import attribute
+from gaphor.core.styling import (
+    Style,
+    TextAlign,
+    VerticalAlign,
+    FontWeight,
+    JustifyContent,
+)
+from gaphor.diagram.presentation import ElementPresentation
+from gaphor.diagram.shapes import (
+    Box,
+    Text,
+    draw_border,
+    draw_left_separator,
+    draw_top_separator,
+    Orientation,
+)
+from gaphor.i18n import gettext
+
+
+class MetadataItem(ElementPresentation):
+    createdBy: attribute[str] = attribute("createdBy", str, "")
+    description: attribute[str] = attribute("description", str, "")
+    website: attribute[str] = attribute("website", str, "")
+    revision: attribute[str] = attribute("revision", str, "")
+    license: attribute[str] = attribute("license", str, "")
+    createdAt: attribute[str] = attribute("createdAt", str, "")
+    updatedAt: attribute[str] = attribute("updatedAt", str, "")
+
+    def __init__(self, diagram, id=None):
+        super().__init__(diagram, id)
+
+        self.watch("createdBy", self.update_shapes).watch(
+            "website", self.update_shapes
+        ).watch("description", self.update_shapes).watch(
+            "revision", self.update_shapes
+        ).watch(
+            "license", self.update_shapes
+        ).watch(
+            "createdAt", self.update_shapes
+        ).watch(
+            "updatedAt", self.update_shapes
+        )
+
+    def update_shapes(self, event=None):
+        group_style: Style = {"justify-content": JustifyContent.STRETCH}
+        box_style: Style = {"padding": (4, 4, 4, 4)}
+        text_style: Style = {
+            "text-align": TextAlign.LEFT,
+            "vertical-align": VerticalAlign.TOP,
+        }
+        heading_style: Style = {
+            **text_style,  # type: ignore[misc]
+            "font-size": "small",
+            "font-weight": FontWeight.BOLD,
+        }
+
+        self.shape = Box(
+            *(
+                [
+                    Box(
+                        *(
+                            [
+                                Box(
+                                    Text(
+                                        text=gettext("created by:"),
```

Comment:
Kind of nit, but I think the metadata would look better if it was in Title Case with each work capitalized.

Reply:
Yeah, or all caps but in a small font like it was in my proposal.
https://github.com/gaphor/gaphor/issues/803#issuecomment-986680585

Reply:
Let's stick with Title Case indeed. It's more consistent with the rest of the app.